Added test data to estore testing database. ABA numbers and Embedded …

ScottPiersall opened the pull request:
…credit card numbers in varchar columns

ScottPiersall committed:
Added test data to estore testing database. ABA numbers and Embedeed credit card numbers in varchar columns

ScottPiersall committed:
Fixed Code Type

ScottPiersall committed:
Fixed missing Parenthesis on toString method for PerformanceSummary

ScottPiersall committed:
Fixed oprand Typos in PCSScan

jordanmcm reviewed (approved):
The adjustments look good to me and the testing data successfully imports if you first make a database ("CREATE DATABASE new_database")

ScottPiersall committed:
Fixed Typo in PCSI Scan Variable

ScottPiersall committed:
Method Invokation fix

ScottPiersall committed:
Invokation spelling

ScottPiersall committed:
Fix

ScottPiersall committed:
Added maria reader connection test. SchemaItem and SechmaItems

ScottPiersall committed:
More changes to RoundTable class

ScottPiersall committed:
Added mising return

ScottPiersall committed:
main return type and values

ScottPiersall committed:
Removed AWT Import

ScottPiersall committed:
removed AWT

ScottPiersall committed:
Removed Desktop AWT Reference

ScottPiersall committed:
Added debug printing lines

ScottPiersall committed:
Adding debug

ScottPiersall committed:
Added PCI Scan Call to Roundtable for testing/debug

ScottPiersall committed:
Added try for scan

jordanmcm committed:
Update HTMLErrorOut.java

This is a non-Linux compatible command and can be used for testing but probably should not be committed

ScottPiersall committed:
Removed Visual Studio Files from Source Tracking

ScottPiersall committed:
Merge branch 'SCOTTPIERSALL_REGEX' of https://github.com/ScottPiersall/RoundTabler into SCOTTPIERSALL_REGEX
